xplique/metrics/fidelity.py

```diff
@@ -407,8 +427,36 @@ def evaluate(self,
         Returns
         -------
         causal_score
-            Metric score, area over the deletion (lower is better) or insertion (higher is
-            better) curve.
+            Metric score (for interpretation, see score interpretation in the documentation).
```

Comment:
It still would be nice to have a summary of the documentation for interpretation